homeassistant/components/just_nimbus/config_flow.py

```diff
@@ -0,0 +1,95 @@
+"""Config flow for Just Nimbus integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import justnimbus
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_CLIENT_ID, CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_validation as cv
+
+from . import CannotConnect, InvalidClientId
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME): cv.string,
```

Comment:
We should not as for a name when setting up the integration. Ideally, we extract the name from the device/service data, OR we use a default name.